.changes/dc281a10-9a1a-4121-b699-1d0ad8271b58.json

```diff
@@ -0,0 +1,5 @@
+{
+    "id": "dc281a10-9a1a-4121-b699-1d0ad8271b58",
+    "type": "misc",
+    "description": "Bump package version to 0.12.4 after broken publish of 0.12.3."
+}
```

Comment:
**Nit**: This isn't the kind of thing I'd expect to see in a changelog. I suggest removing it and making this a **no-changelog** PR.